crates/schema/src/auto_migrate.rs

```diff
@@ -44,16 +45,24 @@ impl<'def> MigratePlan<'def> {
         }
     }
 
-    pub fn pretty_print(&self) -> anyhow::Result<String> {
+    pub fn pretty_print(&self, no_color: bool) -> anyhow::Result<String> {
```

Comment:
```suggestion
    #[derive(Copy, Clone, PartialEq, Eq)]
    pub enum PrettyPrintStyle {
        AnsiColor,
        NoColor,
    }
    pub fn pretty_print(&self, style: PrettyPrintStyle) -> anyhow::Result<String> {
```

Boolean flag arguments like this are difficult to read and interpret. Please use an enum with named variants instead.